### ISSUE TITLE
Field Arrays bug example

### ISSUE THREAD

**Issue by bernabe9** [erikras/redux-form#2581]
### Are you submitting a **bug report** or a **feature request**?
Bug in this [example](http://redux-form.com/6.5.0/examples/fieldArrays/)

### What is the current behavior?
The `members` Fields Array doesn't display the global error(`'At least one member must be entered'`) after submit.

### What is the expected behavior?
It should display the global error after submit.

### What's your environment?
Redux Form version: v6.5.0
OS/browser: Mac OS/ Chrome

### Other informations
In the new release the FieldArray is not touched after submit(https://github.com/erikras/redux-form/pull/2473), this is the reason why is not displaying the global error.
Maybe passing the prop `submitFailed` to the `ConnectedFieldArray` could solve the problem, displaying the error only if there is an error and the form was submitted.

**Comment by tecnolocoEdan** [comment]
Hello, I want to report that this issue is still happening. Touched meta prop is not set to true on submit, also submitFailed is not passed yet. As an example I am attaching the inspection of FieldArray example in docs.

![redux-error](https://cloud.githubusercontent.com/assets/15238607/23472521/b64392ba-fe72-11e6-830e-44dbc55279af.PNG)

**Comment by gustavohenke** [comment]
Hi @tecnolocoEdan, the fix for this was not released yet.

**Comment by tecnolocoEdan** [comment]
I have inspected the code and I found some interesting things:

- The state prop for a ConnectedFieldArray is an Array of its inner Field's states,

![redux-error2](https://cloud.githubusercontent.com/assets/15238607/23486684/2889561e-fea7-11e6-87df-adb836f8e64e.PNG)

as a result on [https://github.com/erikras/redux-form/blob/master/src/createFieldProps.js#L96](url) , you will not be able to get the touched from state.

- On [https://github.com/erikras/redux-form/blob/master/src/reduxForm.js#L413](url) FieldsArray's are taken out from fields list intentionally, so on [https://github.com/erikras/redux-form/blob/master/src/handleSubmit.js#L10](url) they do not get touched on submit.

All in all it  seems that we cannot get FieldArray's touched even with this or the others commits than haven't been released. I want to point out that this fix is not the right solution for the problem because  submitFailed would not return to false if we edit which ever Field after trying to submit and failed.

While inspecting the code and reviewing all of the above, I was thinking on a solution for the touched matter, and the only thing I could think of was to take fields inside a FieldArray into a fields prop. So we get something like:

`
                 fields:{
                    field_array:{
                        fields:[
                            {some_field: 'some value', other_field: 'other value'},
                            {some_field: 'some value', other_field: 'other value'},
                            {some_field: 'some value', other_field: 'other value'}
                        ]
                    }
`
and then in the state in ConnectedFieldArray we can access touched prop if we put it at the same level of fields inside field_array.
